examples/dt/uni04delta/edpm.md

```diff
@@ -0,0 +1,64 @@
+# Deploying the OpenStack dataplane
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been successfully deployed.
+- The [ceph](edpm-ceph.md) has been successfully deployed.
```

Comment:
Perhaps say "The ceph nodes have been deployed and ceph is running on them"

Reply:
Fixed